From e7afacf09d39d7087a5ea112fefabb6d5d3adf2b Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Mon, 9 Dec 2024 18:10:43 +0100
Subject: [PATCH] dts: qcom: add AQR NVMEM node for IPQ9574 RDP433 board

Add Aquantia NVMEM node for IPQ9574 RDP433 board to load the firmware
for the Aquantia PHY.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 arch/arm64/boot/dts/qcom/ipq9574-rdp433.dts | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/arm64/boot/dts/qcom/ipq9574-rdp433.dts
+++ b/arch/arm64/boot/dts/qcom/ipq9574-rdp433.dts
@@ -49,11 +49,17 @@
 	phy4: ethernet-phy@8 {
 		compatible ="ethernet-phy-ieee802.3-c45";
 		reg = <8>;
+
+		nvmem-cells = <&aqr_fw>;
+		nvmem-cell-names = "firmware";
 	};
 
 	phy5: ethernet-phy@0 {
 		compatible ="ethernet-phy-ieee802.3-c45";
 		reg = <0>;
+
+		nvmem-cells = <&aqr_fw>;
+		nvmem-cell-names = "firmware";
 	};
 };
 
